packages/cli/src/commands/source/push/delete.spec.ts

```diff
@@ -0,0 +1,259 @@
+jest.mock('../../../lib/config/config');
+jest.mock('../../../hooks/analytics/analytics');
+jest.mock('../../../hooks/prerun/prerun');
+jest.mock('../../../lib/platform/authenticatedClient');
+jest.mock('@coveo/push-api-client');
+
+import {mocked} from 'ts-jest/utils';
+import {test} from '@oclif/test';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {Source} from '@coveo/push-api-client';
+const mockedClient = mocked(AuthenticatedClient);
+const mockedSource = mocked(Source);
+
+describe('source:push:delete', () => {
+  const doMockAxiosError = (
+    status: number,
+    message: string,
+    errorCode: string
+  ) => ({
+    response: {
+      status,
+      data: {
+        errorCode,
+        message,
+      },
+    },
+  });
+  const doMockAxiosSuccess = (status: number, text: string) => ({
```

Comment:
```suggestion
  });
  
  const doMockAxiosSuccess = (status: number, text: string) => ({
```

watchout the suggestion tho, done from windows, this brk line might be a crlf